crd.Makefile

```diff
@@ -7,7 +7,9 @@ include common.Makefile
 # Installs the application CRD on the cluster.
 .PHONY: crd/install
 crd/install:
-	kubectl apply -f "https://raw.githubusercontent.com/GoogleCloudPlatform/marketplace-k8s-app-tools/master/crd/app-crd.yaml"
+	# Ignore errors on kubectl apply. `AlreadyExists` Errors can occur if
+	# another parallel test is doing a kubectl apply at the same time.
+	-kubectl apply -f "https://raw.githubusercontent.com/GoogleCloudPlatform/marketplace-k8s-app-tools/master/crd/app-crd.yaml"
```

Comment:
I feel like this is a risky move but maybe it will be fine. I'll leave it as-is.

Reply:
Yeah, I did not like adding this, but the first time I ran the 2 test cases in parallel, I saw an `AlreadyExists` error in one test. I didn't expect to see this sort of error when `kubectl apply ...` is being used, but there's apparently a race condition where the 2 instances of `kubectl apply ...` realize that a create is needed, and they both try the create at the same time. It's too bad there's no --ignore-already-created flag or similar to whatever the flag is for `kubectl create ...`.